src/runtime/pkg/katautils/config.go

```diff
@@ -70,8 +70,7 @@ type tomlConfig struct {
 }
 
 type image struct {
-	Provision      string `toml:"provision"`
-	ServiceOffload bool   `toml:"service_offload"`
+	Provision string `toml:"provision"`
 }
```

Comment:
I think we can get rid of the whole `image` struct now we've removed that block from the configuration.toml files?

Reply:
Good point. Removed.